### DISCUSSION TITLE
Lesson 11 Make Mint Error

### DISCUSSION THREAD

**Discussion by truth2012** [Cyfrin/foundry-full-course-cu#1345] [Q&A]
Hi guys, I'm trying to use make mint but this error keeps showing up. Here is the error
truth@DESKTOP-1MUFE0T:~/solidity-course/foundry-f23/foundry-nft-f23$ make mint ARGS="--network sepolia"
[⠊] Compiling...
[⠔] Compiling 1 files with 0.8.23Compiler run successful!
[⠒] Compiling 1 files with 0.8.23
[⠑] Solc 0.8.23 finished in 4.60s
Traces:
  [1131599] → new DevOpsTools@0x2217DfD49033905D06978063f4B3705bbE3D921a
    └─ ← 5710 bytes of code

  [228575] → new MintBasicNft@0x5b73C5498c1E3b4dbA84de0F1833c4a029d90519
    └─ ← 1080 bytes of code

  [6879] MintBasicNft::run()
    ├─ [3925] DevOpsTools::get_most_recent_deployment("BasicNft", 11155111 [1.115e7]) [delegatecall]
    │   ├─ [0] VM::readDir("./broadcast", 3) [staticcall]
    │   │   └─ ← the path broadcast is not allowed to be accessed for read operations
    │   └─ ← the path broadcast is not allowed to be accessed for read operations
    └─ ← the path broadcast is not allowed to be accessed for read operations

Error: 
script failed: the path broadcast is not allowed to be accessed for read operations
make: *** [Makefile:46: mint] Error 1

**Comment by 0xusmanf** [comment]
Maybe [this](https://github.com/Cyfrin/foundry-full-course-f23/discussions/951#discussioncomment-7159919) answer can help.
In your case you should replace `"./img/"` with `"./broadcast"` like in the example below.
```toml
fs_permissions = [{ access = "read", path = "./broadcast"}]
```
